src/styles/index.scss

```
@@ -1,3 +1,148 @@
+h1,
+h2,
+h3,
+h4,
+div,
+body,
+p,
+ul,
+ol {
+  margin: 0;
+}
+
 body {
   margin: 0;
+  background-color: rgb(131, 88, 169);
+  font-family: Roboto, sans-serif;
+
+  --card-width: 200px;
+  --car-height: 408px;
+}
+
+.card {
+  display: block;
+  position: relative;
+  width: var(--card-width);
+  height: var(--car-height);
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: #060b35;
+    margin-bottom: 4;
+  }
+
+  &__description {
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin: 0;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: #060b35;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-title {
+      font-size: 12px;
+      line-height: 18px;
+      color: #616070;
+    }
+
+    &-value {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+      color: #060b35;
+    }
+  }
+
+  &__button {
+    display: flex;
+    box-sizing: border-box;
+    height: 40px;
+    width: 100%;
+    border: 1px solid rgba(0, 172, 220, 1);
+    border-radius: 5px;
+    background-color: rgba(0, 172, 220, 1);
+    font-size: 14px;
+    font-weight: 700;
+    color: white;
+    text-transform: uppercase;
+    text-decoration: none;
+    align-items: center;
+    justify-content: center;
+    margin: 0;
+
+    transition:
+      background-color 0,
+      3s,
+      color 0,
+      3s,
+      border-color 0,
+      3s;
```

Incorrect formatting of the transition property. You must specify each property together with the duration, not separately. Also, unnecessary commas.
```suggestion
    transition:
      background-color 0.3s, 
      color 0.3s, 
      border-color 0.3s;
```